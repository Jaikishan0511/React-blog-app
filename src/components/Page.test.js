import React from "react";
import renderer from "react-test-renderer";

import Page from "./index";
describe("Page", () => {
  test("snapshot renders", () => {
    const component = renderer.create(<Page />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
