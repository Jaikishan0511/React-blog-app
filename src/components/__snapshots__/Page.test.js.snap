// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page snapshot renders 1`] = `
<div>
  <div>
    <div
      className="content"
      id="content"
    >
      <div
        className="row"
      >
        <div
          className="cell"
          id="cell"
          onClick={[Function]}
        />
        <div
          className="cell"
          id="cell"
          onClick={[Function]}
        />
        <div
          className="cell"
          id="cell"
          onClick={[Function]}
        />
      </div>
      <div
        className="row"
      >
        <div
          className="cell"
          id="cell"
          onClick={[Function]}
        />
        <div
          className="cell"
          id="cell"
          onClick={[Function]}
        />
        <div
          className="cell"
          id="cell"
          onClick={[Function]}
        />
      </div>
      <div
        className="row"
      >
        <div
          className="cell"
          id="cell"
          onClick={[Function]}
        />
        <div
          className="cell"
          id="cell"
          onClick={[Function]}
        />
        <div
          className="cell"
          id="cell"
          onClick={[Function]}
        />
      </div>
    </div>
  </div>
</div>
`;
